cirq-core/cirq/transformers/routing/route_circuit_cqc.py

```diff
@@ -15,7 +15,7 @@
 """Heuristic qubit routing algorithm based on arxiv:1902.08091."""
 
 from typing import Any, Dict, List, Optional, Set, Sequence, Tuple, TYPE_CHECKING
-from itertools import combinations
+from itertools import combinations, chain
```

Comment:
make this an absolute import `import itertools` and use `itertools.combinations` and `itertools.chain`